### DISCUSSION TITLE
CSS, enter on value end, can it keep semi-colon and put cursor on new line?

### DISCUSSION THREAD

**Discussion by benfrain** [saghen/blink.cmp#272] [Q&A]
I'm asking the same question here and elsewhere so thought it might be more useful written up as a blog post with some example videos. But in short, can blink.cmp be configured so that pressing enter on a value keeps the semi-colon but moves to a new line?

https://benfrain.com/in-search-of-the-perfect-autocomplete-for-css/

**Comment by saghen** [comment]
Same as hrsh7th's response, this isn't part of autocompletion
